docs/example-narratives/examples/monitoring-set-get.json

```diff
@@ -0,0 +1,37 @@
+{
+  "source": {
+    "last-updated": "2025-10-31T15:05:43.044267100-07:00",
+    "provider": "ISOABC",
```

Comment:
This only appears to be referenced as a GET response example, so consider "RC-NERC-ID" which is used as the value of `authority` in lots of places, since the `ABC` isn't meaningful/necessary to the context.

Reply:
Sure